WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -123,6 +192,24 @@ private extension BlazeCampaignDashboardView {
             .cornerRadius(Layout.cornerRadius)
         }
     }
+
+    func detailView(url: URL) -> some View {
```

Comment:
Super nit: Kindly consider renaming it into `campaignDetailView` for clarity. (To clarify between Product vs Campaign detail)

Reply:
Updated in 5a497604dec9bdfc728b5e71f8a3c6db3ce67015.

Reply:
Done in 5a497604dec9bdfc728b5e71f8a3c6db3ce67015.